.github/workflows/update-deployment-awscli.yml

```diff
@@ -0,0 +1,53 @@
+name: "1. (Internal) Update Deployment - AWS CLI"
+# Create a deployment with a single container with the AWS CLI
+run-name: "${{inputs.target_env}}: Update deployment AWS CLI"
+
+on:
+  workflow_dispatch:
+    inputs:
+      target_env:
+        description: 'Target LHDI environment'
+        required: true
+        default: 'dev'
+        type: choice
+        options:
+        - dev
+        - qa
+        - sandbox
+        - prod-test
+        - prod
+      rollback:
+        description: 'If deploy fails, rollback to previous? (Always true for prod)'
+        required: true
+        type: boolean
+        default: false
+      shutdown:
+        description: '(As a last resort) Shut down chart completely, then redeploy'
+        required: true
+        type: boolean
+        default: false
+
+  # Allow other workflows to call this one
+  workflow_call:
+    inputs:
+      target_env:
+        required: true
+        type: string
+      rollback:
+        required: false
+        type: boolean
+        default: false
+      shutdown:
+        required: false
+        type: boolean
+        default: false
+
+jobs:
+  update-deployment-platform:
+    uses: ./.github/workflows/update-deployment.yml
+    secrets: inherit
+    with:
+      target_env: ${{ inputs.target_env }}
+      helm_chart: awscli
```

Comment:
This won't work unless you also add an `awscli` entry to the options: 
https://github.com/department-of-veterans-affairs/abd-vro/blob/5bb3a5bfd419a91b797e84b6bc440940629db0f9/.github/workflows/update-deployment.yml#L23C1-L23C1